performance/performance-annotation/gradle.properties

```diff
@@ -14,5 +14,5 @@
 # limitations under the License.
 #
 
-artifactRedirecting.publication.targetNames=android
-artifactRedirecting.androidx.groupId=androidx.performance.annotation
\ No newline at end of file
+artifactRedirecting.publication.targetNames=android,iosarm64,iossimulatorarm64,iosx64,jvm,linuxarm64,linuxx64,macosarm64,macosx64,mingwx64,tvosarm64,tvossimulatorarm64,tvosx64,watchosarm32,watchosarm64,watchosdevicearm64,watchossimulatorarm64,watchosx64
```

Comment:
Please note it's published with "jvm" instead of "desktop" suffix for desktop - https://maven.google.com/web/index.html#androidx.performance:performance-annotation-jvm